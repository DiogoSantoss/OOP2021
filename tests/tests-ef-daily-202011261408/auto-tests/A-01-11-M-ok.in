1
app03.dat
3
0
