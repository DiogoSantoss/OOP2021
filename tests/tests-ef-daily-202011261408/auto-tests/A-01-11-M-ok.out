Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Ficheiro a abrir: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Data actual: 10
Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 